core/kazoo_documents/src/kzd_conference.erl

```diff
@@ -0,0 +1,31 @@
+-module(kzd_conference).
```

Comment:
@jamesaimonetti can you check the `cps` branch ? there's a module there's a `kzd_conference` module

Reply:
@lazedo i guess we'll race on getting @k-anderson to merge one or the other first! :)